CMakeLists.txt

```diff
@@ -552,6 +598,7 @@ if(enable-tests)
                 tests/ut_maliit_glib_settings/ut_maliit_glib_settings.c
                 tests/ut_maliit_glib_settings/mockmaliitserver.c
                 tests/ut_maliit_glib_settings/mockmaliitserver.h)
+        set_target_properties(ut_maliit_glib_settings PROPERTIES OUTPUT_NAME "ut_maliit_glib_settings${MALIIT_EXE_SUFFIX}")
```

Comment:
The test also needs to be updated to execute this after rename because it currently fails to run with `Unable to find executable: ut_maliit_glib_settings` (because it's called `ut_maliit_glib_settings6` etc)